### DISCUSSION TITLE
Infinite spinner on login (web3auth.connect())

### DISCUSSION THREAD

**Discussion by nakulthebuilder** [Web3Auth/Web3Auth#968] [Help]
Hi there, had everything working fine (Polygon mainnet, NextJS app) and have changed nothing about the auth implementation, however now all of a sudden whenever i try to run web3auth.connect() am consistently seeing an infinite loader. Logging the web3auth object everything looks fine, with status set to ready. Really feel like i've tried everything and there are no errors being reported so really not sure how to move forward.
Thanks in advance.

**Comment by nakulthebuilder** [comment]
I realized this was more of a nexjs issue and it's working fine now :)
Note - the infinite loading should time out eventually :)

**Comment by speciankarel2-code** [comment]
Pls
